blockapps-strato/azuredeploy.json

```diff
@@ -0,0 +1,231 @@
+                "description" : "Password for admin account 'strato'"
+            }
+        }
+    },
+    "variables": {
+        "imageVHDs": {
+            "live" : "strato-live-image-osDisk.3dfba3b1-de95-49d4-8aaa-ea37a17f5eaa.vhd",
+            "dev"  : "strato-dev-image-osDisk.a136c286-678b-48b9-86b9-56052a28d849.vhd"
+        },
+        "imageVHD": "[variables('imageVHDs')[parameters('blockchain')]]",
+        "imageURI": "[concat('https:\/\/', variables('imageStorage'), '.blob.core.windows.net\/system\/Microsoft.Compute/Images/vhds/', variables('imageVHD'))]",
+        "imageStorage": "stratoimages",
+
+        "name" : "[parameters('instanceName')]",
+        "location": "centralus",
+        "vmSize": "Standard_DS3",
+        "username": "strato",
+        "password": "[parameters('password')]",
+        
+        "storageAccount": "[variables('imageStorage')]",
+        "storageAccountPath": "[concat('Microsoft.Storage/storageAccounts/', variables('storageAccount'))]",
+        "vhdURI": "[concat('https:\/\/', variables('storageAccount'), '.blob.core.windows.net\/vhds\/', parameters('instanceName'), '.vhd')]",
+        "storageAccountType": "Premium_LRS",
+
+        "networkInterfaceName": "[concat(variables('name'), 'nic')]",
+        "networkInterfacePath": "[concat('Microsoft.Network/networkInterfaces/', variables('networkInterfaceName'))]",
+        "networkInterfaceId": "[resourceId('Microsoft.Network/networkInterfaces',variables('networkInterfaceName'))]",
+
+        "networkSecurityGroupName": "[concat(variables('name'), 'nsg')]",
+        "networkSecurityGroupPath": "[concat('Microsoft.Network/networkSecurityGroups/', variables('networkSecurityGroupName'))]",
+        "networkSecurityGroupId": "[resourceId('Microsoft.Network/networkSecurityGroups', variables('networkSecurityGroupName'))]",
+
+        "ipConfigurationName": "[concat(variables('name'), 'ipconfig')]",
+        "publicIPAddressName": "[concat(variables('name'), 'publicip')]",
+        "publicIPAddressPath": "[concat('Microsoft.Network/publicIPAddresses/', variables('publicIPAddressName'))]",
+        "publicIPAddressId" : "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPAddressName'))]",
+        "publicIPAddressType": "Dynamic",
+        "dnsNameForPublicIP": "[variables('name')]",
+
+        "virtualNetworkName": "stratodeploynet",
+        "virtualNetworkPath": "[concat('Microsoft.Network/virtualNetworks/', variables('virtualNetworkName'))]",
+        "vnetId": "[resourceId('Microsoft.Network/virtualNetworks', variables('virtualNetworkName'))]",
+        "virtualNetworkAddressPrefix": "11.24.0.0/16",
+
+        "subnetName": "stratoinstances",
+        "subnetId": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]",
+        "virtualSubnetAddressPrefix": "11.24.82.0/24"
+    },
+    "contentVersion": "1.0.0.0",
+    "$schema": "http:\/\/schema.management.azure.com\/schemas\/2014-04-01-preview\/VM_IP.json"
```

Comment:
Can be replaced with "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#"